test/libsolidity/ABIJson/errors.sol

```diff
@@ -0,0 +1,88 @@
+error FreeError();
+contract X {
+    error E(uint);
+    error E2(uint a, uint b);
+    error E4(uint a, bytes[][] c);
+    struct S { uint x; }
+    error E5(S t);
+    function f() public pure {
+        revert(FreeError());
+    }
+}
```

Comment:
Do inherited errors also show up in the ABI?

What about errors from used libraries?

Reply:
Currently not, I'm waiting for your call graph pr for that.